include/seqan3/argument_parser/validators.hpp

```diff
@@ -451,6 +455,23 @@ class file_validator_base
             return detail::to_string(" Valid file extensions are: [", extensions | views::join(std::string{", "}), "].");
     }
 
+    /*!\brief Helper function that checks if a string is a suffix of another string. Case insensitive.
+     * \param str The string to be searched.
+     * \param suffix The suffix to be searched for.
+     * \returns `true` if `suffix` is a suffix of `str`, otherwise `false`.
+     */
+    bool string_ends_with(std::string_view str, std::string_view suffix) const
```

Comment:
```suggestion
    bool string_view_ends_with(std::string_view str, std::string_view suffix) const
```

Reply:
We could also implement as new std::char_traits that make the string case-insensitive https://en.cppreference.com/w/cpp/string/char_traits